bls/src/main/java/tech/pegasys/teku/bls/BLSSecretKey.java

```diff
@@ -35,10 +66,14 @@ public BLSSecretKey(SecretKey secretKey) {
     this.secretKey = secretKey;
   }
 
-  public SecretKey getSecretKey() {
+  SecretKey getSecretKey() {
     return secretKey;
   }
 
+  public BLSPublicKey toPublicKey() {
+    return new BLSPublicKey(getSecretKey().derivePublicKey());
+  }
+
   public Bytes toBytes() {
     final Bytes bytes = secretKey.toBytes();
     if (bytes.size() == 48) {
```

Comment:
Oops, magic number. My fault. See comment about having a Constants class.

Reply:
Not relevant now